Allow to add annotations to router service in helm chart

prabhu43 opened the pull request:
Fixes #1337

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/fission/fission/1338)
<!-- Reviewable:end -->

life1347 commented on charts/fission-all/Chart.yaml:
```diff
@@ -1,6 +1,6 @@
 name: fission-all
-version: 1.5.0
+version: 1.6.0
 description: Fission is a fast serverless framework for Kubernetes.
```
We will change this when creating a new release, please rollback the changes. (fission-all and fission-core)

vishal-biyani commented on charts/fission-all/Chart.yaml:
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: fission-all
-version: 1.5.0
```
Please don't change the chart version as part of this PR - that will be changed as part of Fission release 1.6

vishal-biyani commented on charts/fission-core/Chart.yaml:
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: fission-core
-version: 1.5.0
```
Please don't change the chart version as part of this PR - that will be changed as part of Fission release 1.6

vishal-biyani commented:
@prabhu43 - thanks for this useful PR. Please address two small issues I have commented on.

prabhu43 commented:
@vishal-biyani @life1347 Thanks for your feedback. I changed the chart version back to 1.5.0 and updated the chart readme file. Also I moved this config to `router.svcAnnotations`

life1347 commented:
LGTM
